templates/repo/issue/branch_selector_field.tmpl

```diff
@@ -1,11 +1,11 @@
 {{if and (not .Issue.IsPull) (not .PageIsComparePull)}}
 <input id="ref_selector" name="ref" type="hidden" value="{{.Issue.Ref}}">
-<input id="editing_mode" name="edit_mode" type="hidden" value="{{(or .IsIssueWriter .HasIssuesOrPullsWritePermission)}}">
```

Comment:
remove ``IsIssueWriter``, because it haven't be defined in any place, I wonder why add it ...

Reply:
It appears you removed it: #10872

Reply:
Oh, I forgot it. So it's okay to remove it.